src/lib/babel-custom.js

```diff
@@ -93,14 +93,18 @@ export default babelPlugin.custom(babelCore => {
 			const babelOptions = config.options || {};
 
 			const envIdx = (babelOptions.presets || []).findIndex(preset =>
-				preset.file.request.includes('@babel/preset-env'),
+				preset.file.request.test(/@babel\/(preset-)?env/),
```

Comment:
this seems to be broken

Reply:
Hmm, I wonder if the previous would've been better